tests/js/asserts.js

```diff
@@ -36,7 +36,9 @@ module.exports = {
             this.assertEqual(val1 && val1.getTime(), val2.getTime(), errorMessage, depth + 1);
         }
         else if (type === 'object') {
-            for (const key of Object.keys(val1)) {
+            var keys = val1.keys !== undefined ? val1.keys() : Object.keys(val1);
```

Comment:
When can it be undefined? Wouldn't that be a bug?

Reply:
The meaning of this change is to refactor the old code which was using `Object.keys`. Now that we have RealmObjects having `keys` function on them this piece of code should check if the object have keys method and use that instead of using Object.keys. For all other objects it should continue using `Object.keys`

Reply:
I'm sorry - I was not paying attention to the filename here. This should of course support asserting stuff about non-Realm objects too :+1: